dashboards/google-microservices/microservices-grpc-monitoring.json

```diff
@@ -43,13 +43,14 @@
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "timeSeriesQueryLanguage": "fetch k8s_container\n  | { t_0:\n      metric 'custom.googleapis.com/opencensus/grpc.io/client/completed_rpcs'\n        | filter (metric.grpc_client_status != 'OK')\n        | align delta(1m)\n        | every 1m\n        | group_by drop [metric.grpc_client_status],\n            [error_rpcs_aggregate: aggregate(value.completed_rpcs)]\n      ; t_1:\n      metric 'custom.googleapis.com/opencensus/grpc.io/client/completed_rpcs'\n        | align delta(1m)\n        | every 1m\n        | group_by drop [metric.grpc_client_status],\n            [completed_rpcs_aggregate: aggregate(value.completed_rpcs)] }\n  | outer_join [0]\n  | value [concurrent_rpcs: div(t_0.error_rpcs_aggregate, t_1.completed_rpcs_aggregate)]\n"
+                  "timeSeriesQueryLanguage": "fetch k8s_container\n  | { t_0:\n      metric 'custom.googleapis.com/opencensus/grpc.io/client/completed_rpcs'\n        | align delta(1m)\n        | every 1m\n        | group_by ignore [metric.grpc_client_status],\n            [completed_rpcs_aggregate: aggregate(value.completed_rpcs)]\n      ; t_1:\n      metric 'custom.googleapis.com/opencensus/grpc.io/client/completed_rpcs'\n        | filter (metric.grpc_client_status != 'OK')\n        | align delta(1m)\n        | every 1m }\n  | outer_join [0]\n  | value [error_rpc_ratio: div(t_1.completed_rpcs, t_0.completed_rpcs_aggregate)]\n"
                 }
               }
             ],
             "thresholds": [],
             "timeshiftDuration": "0s",
             "yAxis": {
+              "label": "y1Axis",
```

Comment:
We recently had to remove all yAxis label definitions as an upcoming change to the charting library will now render them on chart axes. Can you remove all of the label fields in this PR?

Reply:
Done.